### ISSUE TITLE
You are not connected to the TS3 - Question

### ISSUE THREAD

**Issue by IMmmKI** [Newcomer1989/TSN-Ranksystem#384]
Hey, 

I am having an issue with the stats page, when a user goes to the website and tries to assign groups or look at their own stats, they cannot as it says they are not connected to the TS3 server. What could be the issue, it was working a while ago? Is there any way of me troubleshooting the issue? 
They have tried the refresh button. 
I can also see the users IP is correct in the admin view of the users. 

Kind Regards, 

MmmK

**Comment by Shad86** [comment]
This behavior occurs only in 2 cases:

1. if the bot is offline

2. if the user connects to the TS3-Server with IPV6 and to the homepage with ipv4 (the bot cannot match 2 different network technologies) - 

Newcomer is already working on a solution for point 2, but it will take some time - there is currently no eta for it.

**Comment by IMmmKI** [comment]
1. The bot is online. 

2.  I don't see this as an issue for my case at the moment. 

Any other ideas?

**Comment by Shad86** [comment]
They maybe an issue with the session, this can be solved by clicking ont he reload button on the right side:

![session](https://user-images.githubusercontent.com/18570683/34695109-df8b6f00-f4c9-11e7-8fb5-411493ae40f0.png)


> I don't see this as an issue for my case at the moment.

You asked me a question, and I've listed the possible cases when this problem occurs. 

But now you're saying this case doesn't apply to you "right now"?!

So... why? Did you check about it or how did you come to that conclusion?

**Comment by IMmmKI** [comment]
I have tried that a few times and it seems like only after an hour or two does the refresh actually pull through to the website and show the logged in client.

Sorry, seems like you are taking it up the wrong way.  The Ipv6 issue isn't causing the issue right now for the users I am testing with. I know this as the users aren't using Ipv6 and everything is on ipv4. 

Thanks though, I was trying to find more ways to troubleshoot, e.g. what to look for in the logs to see why the refresh isn't updating on the user's end.

**Comment by Newcomer1989** [comment]
> only after an hour or two does the refresh actually pull through to the website and show the logged in client.

But that's looks like point 1) what Shad86 described. You are sure the Bot is online at this moment you are connecting the website? If yes, have you checked the Ranksystem log for errors?
Perhaps you can share this with us for that time, you had tested it the last time.

**Comment by Newcomer1989** [comment]
What you also could test. Go to the "List Rankup" Page and search for your user, when you are online.
Refresh this site after a few seconds and have a look the last seen timestamp will be refreshed with an actual one. If you have not a huge server (thousands of user), it should need at maximum 5 seconds, to get refreshed.

**Comment by Shad86** [comment]
Since there was no further answer, I will close this issue.

If you still have questions, just open a new issue.
